tests/utils.py

```diff
@@ -1,9 +1,10 @@
 import pathlib
 import shutil
 import subprocess
-from typing import Any, Mapping, Sequence
+from typing import Any, Callable, Mapping, Sequence
 
 import pytest
+from librflxlang import AnalysisContext
```

Comment:
While `AnalysisContext` is imported directly from `librflxlang`, other entities of `librflxlang` are imported from `rflx.specification.parser`. I think it makes sense to do the same for `AnalysisContext`.

Reply:
Changed in a496d07a.